applications/padcirc-frontera/profile.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "padcirc",
+    "description": "Modules to load for PADCIRC",
+    "moduleLoads": [
+        {
+            "modulesToLoad": [
+                "intel/19.1.1",
+                "impi/19.0.9",
+                "netcdf/4.6.2",
+                "hdf5/1.10.4",
```

Comment:
Do you happen to know if only padcirc uses the hdf5 package?

Reply:
Ah I didn't notice this differs for PADCIRC + PADCIRC SWAN in DS - I'll assume it is fine to add for PADCIRC SWAN as well.

Reply:
I can't find any documentation about packages to load to run ADCIRC, and the only public app on DS for ADCIRC doesn't load hdf5.